docs/table.md

```diff
@@ -1,144 +1,141 @@
---------------|-----------------------|-----------------|--------------
-`space`       | `m`, `margin`         | `margin`        | `space`
-`space`       | `mt`, `marginTop`     | `margin-top`    | `space`
-`space`       | `mr`, `marginRight`   | `margin-right`  | `space`
-`space`       | `mb`, `marginBottom`  | `margin-bottom` | `space`
-`space`       | `ml`, `marginLeft`    | `margin-left`   | `space`
-`space`       | `mx`                  | `margin-left` and `margin-right`   | `space`
-`space`       | `my`                  | `margin-top` and `margin-bottom`   | `space`
-`space`       | `p`, `padding`        | `padding`       | `space`
-`space`       | `pt`, `paddingTop`    | `padding-top`   | `space`
-`space`       | `pr`, `paddingRight`  | `padding-right` | `space`
-`space`       | `pb`, `paddingBottom` | `padding-bottom` | `space`
-`space`       | `pl`, `paddingLeft`   | `padding-left`  | `space`
-`space`       | `px`                  | `padding-left` and `padding-right`   | `space`
-`space`       | `py`                  | `padding-top` and `padding-bottom`   | `space`
-`width`       | `width`               | `width`         | none
-`fontSize`    | `fontSize`            |`font-size`      |`fontSizes`
-`color`       | `color`               | `color`         | `colors`
-`color`       | `bg`, `backgroundColor` | `background-color`| `colors`
+| Function Name | Prop                    | CSS Property                       | Theme Field |
+| ------------- | ----------------------- | ---------------------------------- | ----------- |
+| `space`       | `m`, `margin`           | `margin`                           | `space`     |
+| `space`       | `mt`, `marginTop`       | `margin-top`                       | `space`     |
+| `space`       | `mr`, `marginRight`     | `margin-right`                     | `space`     |
+| `space`       | `mb`, `marginBottom`    | `margin-bottom`                    | `space`     |
+| `space`       | `ml`, `marginLeft`      | `margin-left`                      | `space`     |
+| `space`       | `mx`                    | `margin-left` and `margin-right`   | `space`     |
+| `space`       | `my`                    | `margin-top` and `margin-bottom`   | `space`     |
+| `space`       | `p`, `padding`          | `padding`                          | `space`     |
+| `space`       | `pt`, `paddingTop`      | `padding-top`                      | `space`     |
+| `space`       | `pr`, `paddingRight`    | `padding-right`                    | `space`     |
+| `space`       | `pb`, `paddingBottom`   | `padding-bottom`                   | `space`     |
+| `space`       | `pl`, `paddingLeft`     | `padding-left`                     | `space`     |
+| `space`       | `px`                    | `padding-left` and `padding-right` | `space`     |
+| `space`       | `py`                    | `padding-top` and `padding-bottom` | `space`     |
+| `width`       | `width`                 | `width`                            | `sizes`     |
```

Comment:
Could probably move `width` down to be included with the other "layout" styles below